devnet-2/droplet-test.md

```diff
@@ -0,0 +1,99 @@
+# Attempt to create network on 3 DO Droplets
+
+Following Instructions from [here](https://github.com/celestiaorg/networks/blob/a378c7cddb91a71db533631d7bbc2b67cb956d5c/README.md)
+
+## Run on all Nodes
+```
+git clone https://github.com/celestiaorg/celestia-app.git
+cd celestia-app 
+apt install make
+cd ..
+wget https://go.dev/dl/go1.17.4.linux-amd64.tar.gz
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.17.4.linux-amd64.tar.gz && rm go1.17.4.linux-amd64.tar.gz
+echo "PATH=$PATH:/usr/local/go/bin" >> ~/.profile
+echo "PATH=$PATH:~/go/bin" >> ~/.profile
+source ~/.profile
+cd celestia-app
+make install
+git clone https://github.com/celestiaorg/networks.git ~/
+cd ../networks/scripts/
+./firewall_ubuntu.sh
+node_name="MightyValidator-${node-num}"
+./1_create_key.sh $node_name
+# Copy Account Address to file
+echo "celes1san3ge6n37jspwsmjl0llh9dveyvylwnrypnff" > ~/account-address-$node_num.txt
+```
+
+## Run on a terminal not ssh'd to a node
+```
+mkdir /tmp/orchestrate && cd /tmp/orchestrate
+node_0_ip=188.166.21.209
+node_1_ip=188.166.91.191
+node_2_ip=174.138.11.26
```

Comment:
Add a comment somewhere that this will require the reader to use their IPs instead (otherwise they will just copy&paste).